app/models/Sprint.php

```diff
@@ -14,6 +15,11 @@ public function project()
 		return $this->belongsTo('Project');
 	}
 
+	public function sprintSnapshots()
```

Comment:
So after quickly looking at this, I guess it returns a `Collection` which contains `SprintSnapshot` instances. Perhaps this is obvious if you know the framework. Still useful for people that don't.